spring-cloud-gcp-docs/src/main/asciidoc/spring-integration-pubsub.adoc

```diff
@@ -0,0 +1,120 @@
+=== Channel Adapters for Google Cloud Pub/Sub
+
+The channel adapters for Google Cloud Pub/Sub connect your Spring
+https://docs.spring.io/spring-integration/reference/html/messaging-channels-section.html#channel[`MessageChannels`]
+to Google Cloud Pub/Sub topics and subscriptions.
+This enables messaging between different processes, applications or micro-services backed up by
+Google Cloud Pub/Sub.
+
+Maven coordinates, using Spring Cloud GCP BOM:
+
+[source,xml]
+----
+<dependency>
+    <groupId>org.springframework.cloud</groupId>
+    <artifactId>spring-integration-gcp</artifactId>
```

Comment:
I think `Pub/Sub` is `optional` now as well and, therefore, deserves some additions here, too.